napari_aicssegmentation/_dock_widget.py

```diff
@@ -14,51 +26,64 @@ class AllenCellStructureSegmenter(QWidget):
+        self.set_page_layout()
+        self.layout().addWidget(self.page)
+
+        # Add scroll widget that holds the page widget
+        scroll = QScrollArea()
+        scroll.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOn)
+        scroll.setWidgetResizable(True)
+        scroll.setWidget(self.page)
+        self.layout().addWidget(scroll)
+
+    """ Add widgets to the page and set the layout """
+    def set_page_layout(self):
+        self.page.setLayout(QVBoxLayout())
 
-        btn = QPushButton("Gaussian kernel size = 3.0")
-        btn.clicked.connect(self.smooth_image)
+        title = QLabel("Segmentation workflow selection")
+        title.setStyleSheet(
+            "QLabel { font-weight: bold; font-size: 20px; margin-top: 0px }")
 
-        desc = QLabel(
-            "Click button to smooth the current viewport image, higher numbers blur "
-            "more. Result is displayed as a new channel."
+        # Need to supply HTML because of this bug: 
+        # https://bugreports.qt.io/browse/QTBUG-90853
+        step_1 = QLabel("<span>1.&nbsp;Select a channel to segment:</span>")
+        dropdown = QComboBox()
+        dropdown.addItem("Channel 1")
+        dropdown.addItem("Channel 2")
+        dropdown.addItem("Channel 3")
+
+        step_2 = QLabel(
+            "<span>2.&nbsp;Select segmentation workflow.&nbsp;"
+            "The image I want to segment most closely resembles "
+            "<b>(click an image)</b>:</span>"
         )
-        desc.setWordWrap(True)
-
-        self.setLayout(QHBoxLayout())
-        self.layout().addWidget(desc)
-        self.layout().addWidget(btn)
-
-    def smooth_image(self):
-        """Guassian Blur on an image, and add it as a new layer"""
-        if self.has_image():
-            # Name to add based off previous image's name
-            name = (
-                self.viewer.layers[len(self.viewer.layers) - 1].name + ": Guassian Blur"
-            )
-
-            # Adding the image to the viewer
-            self.viewer.add_image(
-                image_smoothing_gaussian_3d(self.viewer.layers[0].data, sigma=3.0),
-                name=name,
-            )
-        else:
-            self.show_message_box(
-                "Error: No Image", "Load an image before running guassian blur"
-            )
-
-    def has_image(self):
-        """Determines if there is already an image loaded onto napari"""
-        if len(self.viewer.layers) == 0:
-            return False
-        else:
-            return True
-
-    def show_message_box(self, title, message):
-        """Show a message box with the specified title and message"""
-        msg = QMessageBox()
-        msg.setWindowTitle(title)
-        msg.setText(message)
-        return msg.exec()
+        step_2.setWordWrap(True)
+
+        # This is hacky but not sure if it's worth creating a grid just for this row
+        column_labels = QLabel("Input image                               Segmentation")
```

Comment:
Alternatively, a simple way to this might be to use nested layouts. You can easily achieve 2 columns using a `QHBoxLayout` with 2 labels inside - I believe they will automatically be positioned to take 50% of the layout's width.
See https://realpython.com/python-pyqt-layout/#using-multipage-layouts-and-widgets for a good tutorial on working with layouts